redhat-cop/helm-charts
# fix broken deployment of jenkisn chart with default values

#### What is this PR About?
This fixes the broken deployment of jenkins helm chart with default values - build of all agents fails in init container that performs git clone of agent resources and dockerfile .

![image](https://github.com/user-attachments/assets/e4a26a11-203d-4dc5-bec5-d0d0c7a2cb2e)



#### How do we test this?
The same, just need to deploy the jenkins chart with default values,  and see that all agents are now built successfully.

cc: @redhat-cop/day-in-the-life
